src/twisted/application/reactors.py

```diff
@@ -31,7 +32,7 @@ class IReactorInstaller(Interface):
     """
     )
 
-    def install():
+    def install() -> None:
```

Comment:
does this need a `self`?

Reply:
No, interface definitions do not include `self`.